integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Generic learner data transmitter for integrated channels.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from integrated_channels.integrated_channel.client import IntegratedChannelApiClient
+from integrated_channels.integrated_channel.transmitters import Transmitter
+
+
+class LearnerTransmitter(Transmitter):
+    """
+    A generic learner data transmitter.
+
+    It may be subclassed by specific integrated channel learner data transmitters for
+    each integrated channel's particular learner data transmission requirements and expectations.
+
+    TODO: Find any generic course metadata transmission logic across different integrated channels and put it here.
```

Comment:
@UmanShahzad "Find any generic *learner data* transmission logic..."?

Do you plan on doing this in the scope of the current ticket?

Reply:
Ah yeah, bad copy and paste (again...)

I really would like not to (in the scope of this ticket). Would prefer to just get Degreed Integration going and do any optimizations or remaining refactors later.

Reply:
Fine with me! My main intention was to draw attention to the `TODO`, and to see if you had any additional code coming :)